packages/terra-form-field/src/Field.scss

```diff
@@ -13,10 +13,13 @@
   }
 
   .label-group {
+    font-family: var(--terra-form-field-label-font-family, 'Helvetica Neue');
     font-size: var(--terra-form-field-label-font-size, 1rem);
+    height: var(--terra-form-field-label-height, auto);
```

Comment:
Why are we adding the height and width variables? The height and width will be auto-sized depending on the field control supplied as the child.

Reply:
an attempt to meet this style for [cerner-consumer-theme](https://github.cerner.com/terra/cerner-consumer-theme/pull/1/files#diff-967ea18b58462e0c403edefef7851abfR21) 
![image](https://user-images.githubusercontent.com/6234682/35173494-407302c8-fd31-11e7-9ba1-0136ccf53f1f.png)

I will double check with UX